data/os/CentOS/7.yaml

```diff
@@ -1,3 +1,4 @@
 ---
-letsencrypt::plugin::dns_rfc2136::package_name: 'python2-certbot-dns-rfc2136'
-letsencrypt::plugin::dns_route53::package_name: 'python2-certbot-dns-route53'
+letsencrypt::plugin::dns_rfc2136::package_name: "python2-certbot-dns-rfc2136"
```

Comment:
why did you switch to double quotes? single quotes should be fine

Reply:
Single quotes are fine. My editor just prefers them for some reason (it replaces them on every save operation).

Reply:
I changed my editor's behavior and force pushed.